### ISSUE TITLE
Build problems

### ISSUE THREAD

**Issue by robov** [kristophjunge/docker-mediawiki#16]
When I build your dockerfile I get an error :

Package php-pear is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'php-pear' has no installation candidate

**Comment by robov** [comment]
When i just comment out pear... I get the next errors

```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

**Comment by ghost** [comment]
The dockerfile is intructed to download version ef45039 of VisualEditor. That is:
https://extdist.wmflabs.org/dist/extensions/VisualEditor-REL1_29-ef45039.tar.gz

It should be pointed to b655946, since ef45039 no longer exists:
https://extdist.wmflabs.org/dist/extensions/VisualEditor-REL1_29-b655946.tar.gz

In the dockfile you should change ARG EXTENSION_VISUALEDITOR_VERSION=REL1_29-b655946

**Comment by kristophjunge** [comment]
These were two different problems:

*Installation issues with php-pear*
The underlying PHP base image introduced changes which caused installation issues for php-pear and nginx. This got fixed with commit abfc224.

*Installation issues with extensions*
MediaWiki stores only the latest release of the extensions in the used installation sources. Since newer versions got released it has broken the build. With commit fa73d69 the latest extension release is now determined automatically.
